Utils/WorldZone.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using CitizenFX.Core;
+using CitizenFX.Core.Native;
+
+namespace FivePD-API.Utils

```

Comment:
```suggestion
namespace FivePD.API.Utils
```
As per Microsoft's namespace conventions I would drop the hyphen.

Reply:
I've simply mirrored the current naming convention from the API. This is merely a placeholder until a set up is defined by the devs. But yes, I do agree.